src/pages/Lecture/LectureHeader.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import styled from 'styled-components';
+import { darken, lighten } from 'polished';
+
+function LectureHeader({ title }) {
+  return (
+    <StyledDiv>
+      <Wrapper>
+        <ArrowButton>
+          <i class="fas fa-external-link-alt" />
+          강의 대시보드
+        </ArrowButton>
+        <TitleSpan>{title}</TitleSpan>
+      </Wrapper>
+    </StyledDiv>
+  );
+}
+
+const StyledDiv = styled.div`
+  display: flex;
+  align-items: center;
+  height: 52px;
+  background: #171717;
+  color: #fff;
+
+  @media screen and (max-width: 768px) {
+    display: none !important;
+  }
+`;
+
+const TitleSpan = styled.span`
```

Comment:
```suggestion
const Title = styled.h1 or 2 or 3`
```
그래도 타이틀 내용인데 heading tag가 맞지 않을까 싶어요